internal/commands/user/delete/command.go

```diff
@@ -0,0 +1,81 @@
+package delete
+
+import (
+	"fmt"
+
+	"github.com/10gen/realm-cli/internal/cli"
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+	"github.com/10gen/realm-cli/internal/commands/user/shared"
+	"github.com/10gen/realm-cli/internal/terminal"
+	"github.com/10gen/realm-cli/internal/utils/flags"
+	"github.com/spf13/pflag"
+)
+
+// Command is the `user delete` command
+var Command = cli.CommandDefinition{
```

Comment:
You may have already noticed this, but you will need to add the delete command to the SubCommands section of the user command definition